todo_app/data/mongodb.py

```diff
@@ -0,0 +1,47 @@
+from todo_app.data.classes import to_do_item
```

Comment:
Now that this file exists, it would be good to clear out `trello_items.py` and any other Trello related content since it's no longer relevant

Reply:
Will do